src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -862,8 +862,8 @@ export class SelectBoxList extends Disposable implements ISelectBoxDelegate, ILi
 
 	private updateDetail(selectedIndex: number): void {
 		this.selectionDetailsPane.innerText = '';
-		const description = this.options[selectedIndex].description;
-		const descriptionIsMarkdown = this.options[selectedIndex].descriptionIsMarkdown;
+		const description = (this.options.length > 0) ? this.options[selectedIndex].description : undefined;
```

Comment:
`const description = this.options[selectedIndex]?.description ?? '';` (and same for the other line) - empty string makes more sense I think

Reply:
@smartguest I still think this should use this syntax : 

`const description = this.options[selectedIndex]?.description;`

It's easier to read and does the same thing